jest.config.ts

```diff
@@ -0,0 +1,42 @@
+import type {Config} from '@jest/types';
+
+const config: Config.InitialOptions = {
+  verbose: true,
+  preset: 'ts-jest',
+  coverageDirectory: '__coverage__',
+  collectCoverage: true,
+  transformIgnorePatterns: [
+    'node_modules/(?!@patternfly|lodash-es|@console|@novnc|@spice-project|@popperjs|i18next)'
```

Comment:
I guess this was copied from Console, we can remove some stuff from here.

Reply:
Yup, I think we can remove the `@console`, but I'd rather keep the rest in here. It's always a struggle for new devs to find why some new tests are failing because they are testing for instance a component that imports `@patternfly`.